src/common/routes/Routes.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
```

Comment:
We shouldn't have this

Reply:
I agree, but this is only for the line below with `any`, so will be removed after redux flow implementation
`any // remove this any after user login is taken by redux`

Reply:
I agree, but this is only for the line below with `any`, so will be removed after redux flow implementation
`any // remove this any after user login is taken by redux`

Reply:
In this case, could we use `// eslint-disable-next-line @typescript-eslint/no-explicit-any`? That would disable the rule only for the line where it should be ingored

Reply:
Sure, I just pushed changes. Unfortunately, merging master failed, because of other files. Let me know if you know why it happened.

![Screenshot 2021-05-26 at 07 59 11](https://user-images.githubusercontent.com/81692167/119609676-46a9bc00-bdf8-11eb-9ba5-15018ff31362.png)